services/app-api/src/handlers/postgres_migrate.ts

```diff
@@ -101,8 +97,14 @@ export const main: APIGatewayProxyHandler = async () => {
 
     // run the data migration. this will run any data changes to the protobufs stored in postgres
     try {
-        const migrator = newDBMigrator(dbConnectionURL)
-        await migrate(migrator, '/opt/nodejs/')
+        execSync(
```

Comment:
just checking my understanding - we are not able to use MacRae's fav pattern of dependency injection anymore here _because_ it would require that cross service import to get the protos. Is that right? This seems like the area of our code in this fix directly related to https://qmacbis.atlassian.net/browse/MR-2671

Reply:
yeah, I had to move away from the pattern in the handler itself because of issues from the interplay between the lambda environment, the webpack bundle, and the import of compiled scripts from `app-proto`.